app/routers/organizations.py

```diff
@@ -1,15 +1,43 @@
 from fastapi import APIRouter, status, HTTPException
 from database import client
 from models import Organization
+from fastapi.responses import JSONResponse
+import string
+import random
 
 router = APIRouter()
 
 
+@router.post("/create-org/{org_name}")
+async def create_organization(org_name: str):
+    if org_name is not None:
+        # create an API key
+        char_set = string.ascii_letters + string.punctuation
+        urand = random.SystemRandom()
+        org_key = "".join([urand.choice(char_set) for _ in range(20)])
+
+        # check if API key exists
+        if (client.quiz.organization.find_one({"org_key": org_key})) is None:
```

Comment:
You'll also have to define `org_key` in the model as well right?
Currently the model only has `org_name`

Reply:
Response shouldn't have key, no?